karaf/init/pom.xml

```diff
@@ -67,9 +53,7 @@
                         <Bundle-ClassPath>.,_brooklyn-tosca-karaf-patches-${project.version}.jar</Bundle-ClassPath>
             <Import-Package>org.apache.brooklyn.api.*</Import-Package>
             <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
aha i see, this dynamic-import is able to pull what we need from brooklyn while ignoring those deps which are embedded.  i'm a little nervous about this approach as compared with the explicit require-bundle but it sounds reasonable.

maybe worth checking the wiring at runtime to make sure it isn't wiring in things we don't want (e.g. not wired to things which come from embedded dependencies).